doc/Command-Reference.md

```diff
@@ -2417,6 +2417,22 @@ Dynamic breakout feature is yet to be supported in SONiC and hence uses cannot c
   admin@sonic:~$ sudo config interface Ethernet63 speed 40000
   ```
 
+**config interface mtu <interface_name> (Versions >= 201904)**
+
+This command is used to configure the mtu for the Physical interface. Use the value 1500 for setting max transfer unit size to 1500 bytes.
+
+- Usage:
+
+  *Versions >= 201904*
+  ```
+  config interface mtu <interface_name> <mtu_value>
```

Comment:
Please prefix with `sudo`.

Reply:
The description of the command was done by analogy with the other interface actions provided in the doc, i.e. **shutdown**, **startup**, **speed**. The change will contradict the commonly accepted description convention.

Reply:
@jleveque do you agree with the explanation?

Reply:
My point is the lines are inconsistent. Some have the `admin@sonic:~$` prompt, some do not. I would like them all consistent (this also applies to some of the existing lines, as well; those will need to be handled separately.